sort/js/search.js

```diff
@@ -0,0 +1,44 @@
+function getArray(size) {
+  var RANDOM_STEP = 8;
+  var array = [size];
+  array[0] = Math.floor(Math.random() * (-size * RANDOM_STEP >> 2));
+  for (var i = 1; i < size; i++) {
+    array[i] = array[i - 1] + Math.floor(Math.random() * RANDOM_STEP + 1);
+  }
+  return array;
+}
+
+function isSorted(items) {
```

Comment:
`getArray` and `isSorted` are kind of helpers you use in tests, but you don't use it in your methods
imo they should be moved to test file